src/tools/include/cct_hash.hpp

```diff
@@ -42,13 +42,13 @@ namespace std {
 template <class... TT>
 struct hash<std::tuple<TT...>> {
   size_t operator()(const std::tuple<TT...>& tup) const {
-    auto lazyHasher = [](size_t hash, auto&&... values) {
```

Comment:
Honestly, I don't know what happened here in MSVC. I suspect a conflict between variable name `hash` and function `std::hash` but it's not in the same namespace...